.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,37 @@
+<!-- 
+Please explain WHAT you changed and WHY. 
+
+The title should be descriptive, for example:
+
+* *Fixed a typo in the apikeypermissions.md page*
+* *Added the maximum number of domain whitelabels you can create to domains.md*
+* *Fixing the number of days a batch id is valid in scheduling_parameters.md*
+
+If this PR fixes an issue, please reference the issue number as well.
+
+Fill out this form in the body:
+-->
+
+**Description of the change**:
+
+**Reason for the change**:
+
+**Link to original source**:
+
+### Checklist
+
+Make sure all of these items are complete, or else the PR will be ineligible for a code review.
+
+- [ ] Code passes all existing [tests](https://github.com/sendgrid/nodejs-http-client/tree/master/test)
+- [ ] Any new functionality added includes new unit tests in [`test/test.js`](https://github.com/sendgrid/nodejs-http-client/tree/master/test/test.js)
+- [ ] Create or update example code to show the new functionality in action.
+- [ ] All code, branch, and git naming and style conventions are followed (see [`CONTRIBUTING.md`](https://github.com/sendgrid/nodejs-http-client/blob/master/CONTRIBUTING.md#style-guidelines--naming-conventions))
+- [ ] Feature branch has been rebased off of the latest `master` branch. ( see [`CONTRIBUTING.md`](https://github.com/sendgrid/nodejs-http-client/blob/master/CONTRIBUTING.md#creating-a-pull-request) ).
+
+If you have questions, please send an email [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.
+
+<!-- 
```

Comment:
Can you please remove this section? This is linking back to sendgrid/docs and may confuse people. I removed it from the source file too - sorry about that!